api4/webhook_test.go

```diff
@@ -135,17 +126,11 @@ func TestGetIncomingWebhooks(t *testing.T) {
 			found = true
 		}
 	}
-
-	if !found {
-		t.Fatal("missing hook")
-	}
+	require.True(t, found, "missing hook")
 
 	hooks, resp = th.SystemAdminClient.GetIncomingWebhooks(0, 1, "")
 	CheckNoError(t, resp)
-
-	if len(hooks) != 1 {
-		t.Fatal("should only be 1")
-	}
+	require.Equal(t, len(hooks), 1, "should only be 1")
```

Comment:
Awesome, thanks @drekar !
This should also be `expected, actual`. And a few more like this below. 
Other than that, looking good, almost ready.